client/claims.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 OpenPubkey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package client
```

Comment:
I often struggle to remember that the interfaces are in this file with a bunch of other seemingly random things. I moved the relevant interfaces to the client file because they were significantly changed and it was a good opportunity to put them in the correct place (according to Go). I would love to delete the file entirely but I kept it around for legacy purposes as it doesn't contain any code that we use in our library.

I think it is important to also prioritize new developers coming in and where they might look for something like this. Someone who is not constantly touching the code may likely go to default expected locations (like me) as opposed to where they "remember" it being.